transcendence-app/src/user/user.domain.ts

```diff
@@ -8,8 +9,8 @@ export class User {
   @Exclude()
   password!: string | null;
   avatarId!: string | null;
-  avatarX = 0;
-  avatarY = 0;
+  avatarX: number = 0;
+  avatarY: number = 0;
```

Comment:
The OpenAPI generator needs these types, even if TypeScript could infer them.
I've removed them in a previous PR :sweat_smile: